network/roles/google_cloud_vpc/defaults/main.yaml

```diff
@@ -0,0 +1,16 @@
+---
+state: present
+
+id: "{{ omit | hash('sha1') }}"
+
+gcp_network_name: "vpc-{{ id[0:6] }}"
+gcp_network_routing_mode: REGIONAL
+gcp_network_auto_create_subnets: True
+gcp_network_cidr: null
```

Comment:
Didn't see where `gcp_network_cidr` is being used, wondering if it can be removed?